theano/tensor/tests/test_opt.py

```diff
@@ -2,8 +2,6 @@
 
```

Comment:
That file contains pickled CudaNdArrayType.  So it doesn't work when you remove the old backend.